lib/tags/group-creator.tag

```diff
@@ -145,8 +147,10 @@
                             }
                         } else {
                             self.refs.title.classList.remove('graphjs-success');
+                            self.refs.title.removeAttribute('data-savesuccesstext');
```

Comment:
Please remove this change. self.refs.title refers to an input element and input elements do not accept ::before or ::after pseudo-elements.

Reply:
Same as above